packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -218,6 +215,7 @@ export default function MainFeedLayout({
     }
 
     if (isSearchOn && searchQuery) {
+      const searchVersion = getFeatureValue(feature.searchVersion);
```

Comment:
Just curiosity why are only getting the version here? or we only need it here?

Reply:
Yeah only needed here,
Was a edge case where the other hook would still be loading and return the default value and then never retrigger.